svc-lighthouse-api/src/main/java/gov/va/vro/abddataaccess/model/AbdMedication.java

```diff
@@ -16,6 +16,8 @@ public class AbdMedication implements Comparable<AbdMedication> {
   private String description;
   private int refills;
   private Boolean asthmaRelevant;
```

Comment:
SO what are we doing with asthmaRelevant?

Reply:
It should be removed when it is safe to do so.